tcf_website/static/reviews/sync_review_sliders.js

```diff
@@ -0,0 +1,15 @@
+/* Sync Review Rating Slider InputIds */
+jQuery(($) => {
+    const tuples = [
+        ["#instructor_rating", "#instructor_rating2"],
+        ["#enjoyability", "#enjoyability2"],
+        ["#difficulty", "#difficulty2"],
+        ["#recommendability", "#recommendability2"]
+    ];
+
+    tuples.forEach((tuple) => {

```

Comment:
Nice refactor, but perhaps add some in-text comments since it's a little less clear what the code is doing now

Reply:
Added a couple lines of comments.